tokio/src/lib.rs

```diff
@@ -339,14 +339,32 @@
 //! _parking_lot_ release in use.
 //!
 //! ### Unstable features
-//!
-//! These feature flags enable **unstable** features. The public API may break in 1.x
-//! releases. To enable these features, the `--cfg tokio_unstable` must be passed to
-//! `rustc` when compiling. This is easiest done using the `RUSTFLAGS` env variable:
-//! `RUSTFLAGS="--cfg tokio_unstable"`.
-//!
+//! 
+//! Some feature flags are only available when specifying the `tokio_unstable` flag:
+//! 
 //! - `tracing`: Enables tracing events.
+//! 
+//! Likewise, some parts of the API are only available with the same flag:
+//! 
+//! - [`task::JoinSet`]
+//!  
+//! This flag enables **unstable** features. The public API of these features
+//! may break in 1.x releases.
+//! 
+//! To enable these features, the `--cfg tokio_unstable` must be passed to
+//! `rustc` when compiling. This is easiest done using the `RUSTFLAGS` env
+//! variable: `RUSTFLAGS="--cfg tokio_unstable"`.
```

Comment:
This should probably show a full invocation to `cargo build`. Preferably for both unix and windows if different.

Reply:
This blurb was previously existing (from 79d25b0a48d3e82a9d026e9cdfeef548153660ef) and just got reformatted with my changes.   I just left it in as-is.

I'm wary about trying to explain how to use environment variables and `cargo build` on various systems, though.  (For example, on Windows it's different depending on whether you're using the command prompt or PowerShell. Not to mention the various *nix shells.)

Reply:
You don't have to document every possible shell out there, but I still think that including some common variants is useful.

Reply:
OK, added *nix + Windows PowerShell examples.